### PR TITLE
Updates to the installation docs

### PR DESCRIPTION
**Pull request by thewtex**
Recommend pip over easy_install.  Give instructions for installation from source.

The `pip install` command will not work until an initial package is on PyPI.

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -2,11 +2,36 @@
...
+    $ pip install scikit-build
 
-Or, if you have virtualenvwrapper installed::
+From source::
 
-    $ mkvirtualenv scikit-build
-    $ pip install scikit-build
+    $ git clone https://github.com/scikit-build/scikit-build
+    $ cd scikit-build
+    $ python setup.py install
```
Comment (jcfr) [inline docs/installation.rst]:
```
$ pip install .
```

Comment (jcfr) [inline docs/installation.rst]: or 

```
pip install -e .
```

for development

**Comment by jcfr** [comment]
:+1: Looks great. Thanks

File: docs/installation.rst (hunk 1 above)
Comment (thewtex) [inline docs/installation.rst]: :+1: Done.

File: docs/installation.rst [hunk 2]
```diff
@@ -2,11 +2,45 @@
...
+
+Dependencies
+------------
+
+The project has a few common Python package dependencies. The runtime
+dependencies are:
+
+.. include:: ../requirements.txt
+
+the build time dependencies are:
```
Comment (jcfr) [inline docs/installation.rst]: the build time dependencies (also required for development) are:

Comment (thewtex) [inline docs/installation.rst]: Done.

**Comment by opadron** [comment]
LGTM

**Comment by jcfr** [comment]
Gracias muchos

**Pull request merged**